embed/src/app.py

```diff
@@ -48,18 +51,21 @@ def embed_queue_item(self, queue_item: EmbedQueueItem):
         """
 
         chunk: Chunk = self.storage_handler.read_item(queue_item.chunk_uuid, "Chunk")
-        embedded_sentences = embed_sentences(queue_item.model, [chunk.text])
+        embedded_sentences = model_db.embed_sentences(queue_item.model, [chunk.text])
         if len(embedded_sentences.data) != 1:
-            raise ValueError(f"expected just 1 embedding but got {len(embedded_sentences.data)}")
+            logging.error(f"expected just 1 embedding but got {len(embedded_sentences.data)}")
```

Comment:
I'm not too familiar (at all) with how embeddings work. Would having <1 or >1 result here be a critical issue with the program or just a case of "whoopsie that wasn't right"?

Reply:
the [SentenceTransformer.encode](https://www.sbert.net/docs/package_reference/SentenceTransformer.html#sentence_transformers.SentenceTransformer.encode) always returns a list of embeddings, given that we have only entered one item of text i am checking that only one embedding is returned

Reply:
ok, in that case I think the way you've done it with a logger instead of causing an exception is the right way to go 👍